interop-scoring/main.js

```diff
@@ -0,0 +1,506 @@
+  '/pointerevents/pointerevent_movementxy.html?mouse',
+  '/pointerevents/pointerevent_pointercapture_in_frame.html?mouse',
+  '/uievents/mouse/attributes.html',
+  // interop-2023-webcodecs
+  '/webcodecs/videoDecoder-codec-specific.https.any.html?av1',
+  '/webcodecs/videoDecoder-codec-specific.https.any.html?h264_annexb',
+  '/webcodecs/videoDecoder-codec-specific.https.any.html?h264_avc',
+  '/webcodecs/videoDecoder-codec-specific.https.any.html?vp8',
+  '/webcodecs/videoDecoder-codec-specific.https.any.html?vp9',
+  '/webcodecs/videoDecoder-codec-specific.https.any.worker.html?av1',
+  '/webcodecs/videoDecoder-codec-specific.https.any.worker.html?h264_annexb',
+  '/webcodecs/videoDecoder-codec-specific.https.any.worker.html?h264_avc',
+  '/webcodecs/videoDecoder-codec-specific.https.any.worker.html?vp8',
+  '/webcodecs/videoDecoder-codec-specific.https.any.worker.html?vp9',
+  '/webcodecs/videoDecoder-codec-specific.https.any.worker.html?av1',
+  '/webcodecs/videoFrame-construction.any.html',
+  '/webcodecs/videoFrame-construction.crossOriginSource.sub.html',
+  '/webcodecs/videoFrame-construction.window.html',
+  '/webcodecs/videoFrame-serialization.crossAgentCluster.https.html',
+  '/webcodecs/videoFrame-serialization.crossAgentCluster.https.html',
+  '/webcodecs/temporal-svc-encoding.https.any.html?h264',
+  '/webcodecs/temporal-svc-encoding.https.any.html?vp8',
+  '/webcodecs/temporal-svc-encoding.https.any.html?vp9',
+  '/webcodecs/temporal-svc-encoding.https.any.worker.html?h264',
+  '/webcodecs/temporal-svc-encoding.https.any.worker.html?vp8',
+  '/webcodecs/temporal-svc-encoding.https.any.worker.html?vp9',
+  '/webcodecs/videoFrame-serialization.crossAgentCluster.https.html',
+  '/webcodecs/videoFrame-serialization.crossAgentCluster.https.html',
+  '/webcodecs/videoFrame-serialization.crossAgentCluster.https.html',
+  '/webcodecs/full-cycle-test.https.any.html?av1',
+  '/webcodecs/full-cycle-test.https.any.html?h264_annexb',
+  '/webcodecs/full-cycle-test.https.any.html?h264_avc',
+  '/webcodecs/full-cycle-test.https.any.html?vp9_p0',
+  '/webcodecs/full-cycle-test.https.any.html?vp9_p2',
+  '/webcodecs/full-cycle-test.https.any.worker.html?av1',
+  '/webcodecs/full-cycle-test.https.any.worker.html?h264_annexb',
+  '/webcodecs/full-cycle-test.https.any.worker.html?h264_avc',
+  '/webcodecs/full-cycle-test.https.any.worker.html?vp9_p0',
+  '/webcodecs/full-cycle-test.https.any.worker.html?vp9_p2',
+  '/webcodecs/full-cycle-test.https.any.html?vp8',
+  '/webcodecs/full-cycle-test.https.any.worker.html?vp8',
+  // interop-2023-webcomponents
+  '/shadow-dom/focus/focus-shadowhost-display-none.html',
+  '/custom-elements/form-associated/ElementInternals-labels.html',
+  '/custom-elements/form-associated/ElementInternals-setFormValue.html',
+  '/custom-elements/form-associated/ElementInternals-validation.html',
+  '/custom-elements/form-associated/form-disabled-callback.html',
```

Comment:
No investigations to rectify these non-OK statuses have occurred as of today, and they have been simply added to this list to bypass script failures for the time being. It is unknown to me if action needs to be taken.